src/current/molt/molt-fetch.md

```diff
@@ -503,6 +504,90 @@ The JSON is formatted as follows:
+	- `column` specifies source columns to exclude from being mapped to regular columns on the target schema. It is formatted as a POSIX regex string.
+	- `add_computed_def`, when set to `true`, specifies that each matching `column` should be mapped to a [computed column]({% link {{ site.current_cloud_version }}/computed-columns.md %}) on the target schema, using the computed column definition on the source schema. This assumes that all matching columns are computed columns on the source.
+		{{site.data.alerts.callout_danger}}
+		Columns that match the `column` regex will **not** be moved to CockroachDB if either `add_computed_def` is omitted, or a matching column is a non-computed column.
+		{{site.data.alerts.end}}
+- `table_rename_opts` configures the following option for table renaming:
+	- `value` specifies the table name to which the matching `resource_specifier` is mapped. If there is only one table matching `resource_specifier` on the source, it is renamed to `table_rename_opts.value` on the target. If there is more than one matching `resource_specifier` (i.e., an n-to-1 mapping), this assumes that all matching tables are [partitioned tables]({% link {{ site.current_cloud_version }}/partitioning.md %}) with the same schema. Otherwise, the process will error. 
+	- Additionally, in such `n` to `1` mapping situation: 
+	     - Since these tables' data will be ingested into the single target table _concurrently_, only `--use-copy` or `--direct-copy` mode is allowed for the data ingestion phase. 
+	     - User will have to create the schema of the target table on their own, and must choose `--table-handling=none` mode. I.e. `--table-handling={truncate|drop-on-target-and-recreate}` are disabled for this case.
+
+The preceding JSON example therefore defines two rules:
+
+- Rule `1` maps all source `age` columns on the source database to [computed columns]({% link {{ site.current_cloud_version }}/computed-columns.md %}) on CockroachDB. This assumes that all matching `age` columns are defined as computed columns on the source.
+- Rule `2` maps all table names with prefix `charges_part` from the source database to a single `charges` table on CockroachDB (i.e., an n-to-1 mapping). This assumes that all matching `charges_part.*` tables have the same schema.
+
+Each rule is applied in the order it is defined. If two rules overlap, the later rule will override the earlier rule.
+
+To verify that the logging shoes that the computed columns are being created:
+
+When running `molt fetch`, set `--logging 'debug'` and look for `ALTER TABLE ... ADD COLUMN` statements with the `STORED` or `VIRTUAL` keywords in the log output:
+
+~~~ json
+{"level":"debug","time":"2024-07-22T12:01:51-04:00","message":"running: ALTER TABLE IF EXISTS public.computed ADD COLUMN computed_col INT8 NOT NULL AS ((col1 + col2)) STORED"}
+~~~
+
+After running `molt fetch`, show the `CREATE` statement for a table:
```

Comment:
Ensure that this statement is run on the target. For a sanity check, they can also run `SELECT * from public.computed LIMIT 1`